tests/VarExporterTest.php

```diff
@@ -151,6 +151,24 @@ public function testInlineNumericScalarArray()
         $this->assertExportEquals($expected, $var, VarExporter::INLINE_NUMERIC_SCALAR_ARRAY);
     }
 
+    public function testExportDateTime()
+    {
+        $timezone = new \DateTimeZone('Europe/Berlin');
+        $format = 'Y-m-d H:i:s.u';
+
+        $var = \DateTime::createFromFormat($format, '2020-03-09 18:51:23.000000', $timezone);
+
+        $expected = <<<'PHP'
+\DateTime::__set_state([
+    'date' => '2020-03-09 18:51:23.000000',
+    'timezone_type' => 3,
+    'timezone' => 'Europe/Berlin'
+])
+PHP;
+
+        $this->assertExportEquals($expected, $var);
+    }
+
```

Comment:
Can you please add a test for `DateTimeImmutable` as well?

Reply:
I could do that, but considering my comment above, there could be many more internal classes implementing `__set_state()` without anyone even knowing about that.

I'll add the test for `DateTimeImmutable`, but I don't think that every internal class with that method needs its own test case.